pkg/nodelabels/builder.go

```diff
@@ -76,3 +76,13 @@ func BuildNodeLabels(cluster *kops.Cluster, instanceGroup *kops.InstanceGroup) m
 
 	return nodeLabels
 }
+
+// BuildMandatoryControlPlaneLabels returns the list of labels all CP nodes must have
+func BuildMandatoryControlPlaneLabels() map[string]string {
+	nodeLabels := make(map[string]string)
+	nodeLabels[RoleLabelMaster16] = ""
+	nodeLabels[RoleLabelControlPlane20] = ""
+	nodeLabels[RoleLabelName15] = RoleMasterLabelValue15
+	nodeLabels["kops.k8s.io/kops-controller-pki"] = ""
```

Comment:
I have some concern that this should be only for k8s 1.19+. Am I right?

Reply:
That label is installed unconditionally on any _kOps_ 1.19+ cluster. See https://github.com/kubernetes/kops/pull/10780/files#diff-dacb133dab486c89cbabcce60635171df5295d7058d4b12565787bbfd7d46c0cL51

Reply:
OK, let's see how this goes.